Assignment2/adrii0912-fromJSONtoJSON-LD.jsonld

```diff
@@ -0,0 +1,42 @@
+{
+	"@context":{
+		
+		"xsd": "http://www.w3.org/2001/XMLSchema#",
+		"student": "http://www.assignment2.com/student",
+		"fullName":"student:fullName",
+		"hometown":"student:hometown",
+		"age":"student:age",
+		"almaMater":"student:almaMater",
+		"parentsName":"student:parentsName",
+		"subjects":"student:subjects",
+		"subjectName":"subjects:subjectName",
+		"teachers":"subjects:teachers",
+		"description":"subject:description"
+			
+	},
+	
+	"@id":"student",

```

Comment:
Add at the end colon of student if you want to reuse the URI defined in @context.